WonderBewegung.py

```diff
@@ -0,0 +1,261 @@
+            elif "süd" in vergleichsergebnis:
+                if "süd" in benutzereingabe:
+                    print("Du kannst nicht nach Süden " + benutzerWort.gebeBezeichnung() + ".")
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                elif "nord" in benutzereingabe:
+                     self.bewegeEbene(benutzerWort)                    
+                else:
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                    print("Wenn Du nicht nach Süden weiter willst, musst du erst wieder einen Schritt zurück nach Norden laufen!")                    
+            elif "ost" in vergleichsergebnis:
+                if "ost" in benutzereingabe:
+                    print("Du kannst nicht nach Osten " + benutzerWort.gebeBezeichnung() + ".")
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                elif "west" in benutzereingabe:
+                    self.bewegeEbene(benutzerWort)
+                    
+                else:
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                    print("Wenn Du nicht nach Osten weiter willst, musst du erst wieder einen Schritt zurück nach Westen laufen!")                    
+            elif "west" in vergleichsergebnis:
+                if "west" in benutzereingabe:
+                    print("Du kannst nicht nach Westen " + benutzerWort.gebeBezeichnung() + ".")
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                elif "ost" in benutzereingabe:
+                     self.bewegeEbene(benutzerWort)
+                    
+                else:
+                    print(self.umgebung.gebeUebergangssatz(vergleichsergebnis,uebergangstyp))
+                    print("Wenn Du nicht nach Westen weiter willst, musst du erst wieder einen Schritt zurück nach Osten laufen!")                    
+            else: 
+                 self.bewegeEbene(benutzerWort)
+
+            return False  
+
+    def bewege(self, eingabe, bewegungsWort:Wort):
+        self.eingabe =  eingabe
+        
+        self.uebergang = self.pruefeUebergang()
+        self.wonderText.setzeText("")
+
+        if  self.uebergang == False:    
+            wort = self.umgebung.vergleicheWorte(eingabe)
+            if wort is None:
+                self.wonderText.ergaenzeText("Achtung,in diesem Gebiet (" + str(self.umgebung.gebeBezeichnung())+ ") kann man nicht " + bewegungsWort.gebeBezeichnung() + ".")
+            elif "kleineres" not in self.grenzTest:
+                    self.wonderText.ergaenzeText ( self.umgebung.gebeUebergangssatz(self.grenzTest, self.uebergangstyp))                 
+                    self.uebergang = True                                 
+            elif "kleineres" in self.grenzTest:
+                # Bewegung durchführen
```

Comment:
Ich würde an der Stelle bei allen Fällen hinschreiben was das für eine Situation ist ("Spieler*in am Übergang"/"Spieler*in in einem kleinen Abstand vom Übergang"/..). Ich blick das grad zb nicht so ganz durch^^.